README.md

```diff
@@ -47,19 +47,19 @@ Prerequisites: [Node.js](https://nodejs.org/) (`^12.22.0`, `^14.17.0`, or `>=16.
 
 You can install ESLint using npm:
 
-```sh
+```shell
```

Comment:
Is there any benefit from using `shell` instead of `sh` here?

Reply:
It's the same thing but I switched this to `shell` for consistency with all other 90 usages in the codebase.